src/NServiceBus.AcceptanceTests/Core/Installers/When_creating_queues.cs

```diff
@@ -0,0 +1,105 @@
+﻿namespace NServiceBus.AcceptanceTests.Core.Installers
+{
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using Configuration.AdvancedExtensibility;
+    using EndpointTemplates;
+    using FakeTransport;
+    using NUnit.Framework;
+
+    public class When_creating_queues : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public async Task Should_not_create_queues_queue_creation_disabled_and_installers_disabled()
```

Comment:
the test names could use a `when` in their naming, e.g. `Should_not_create_queues_when_queue_creation_disabled_and_installers_disabled`

Reply:
Good catch! Fixed.